BDHelloPointSDk/Info.plist

```diff
@@ -31,6 +31,8 @@
 	<key>UIBackgroundModes</key>
 	<array>
 		<string>location</string>
+		<string>processing</string>
```

Comment:
Why do we require background processing?

Reply:
I think its required for receiving pushes while app is in background